sentry-core/src/main/java/io/sentry/core/transport/HttpTransport.java

```diff
@@ -89,6 +79,12 @@ public TransportResult send(SentryEvent event) throws IOException {
     connection.setRequestMethod("POST");
     connection.setDoOutput(true);
     connection.setRequestProperty("Content-Encoding", "UTF-8");
+    connection.setRequestProperty("Content-Type", "application/json");
+    connection.setRequestProperty("Accept", "application/json");
```

Comment:
@bruno-garcia it works without that, but it's very common to set anyway

Reply:
I don't set in .NET, this prob came from `sentry-java`.
No opinion here, could probably get away without it as u suggest.
@HazAT thoughts?